README.md

```diff
@@ -19,6 +17,58 @@ features:
 For more details, please see the
 [documentation](https://kubernetes-csi.github.io/docs/Setup.html#csidriver-custom-resource-alpha).
 
+## Compatibility
+
+| Latest stable release                                                                                       | Branch                                                                                  | Compatible with CSI Version                                                                | Container Image                                 | Min K8s Version | Max K8s Version |
+| ----------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------ | ----------------------------------------------- | --------------- | --------------- |
+| [cluster-driver-registrar v1.0.1](https://github.com/kubernetes-csi/cluster-driver-registrar/releases/tag/v1.0.1) | [release-1.0](https://github.com/kubernetes-csi/cluster-driver-registrar/tree/release-1.0) | [CSI Spec v1.0.0](https://github.com/container-storage-interface/spec/releases/tag/v1.0.0) | quay.io/k8scsi/csi-cluster-driver-registrar:v1.0.1 | 1.13            | -               |
+
+## Usage
+
+A UNIX domain socket is used by the cluster-driver-registrar:
+
+* CSI driver socket:
+  * Used by kubelet to interact with the CSI driver.
+  * Created by the CSI driver.
+  * Exposed on a Kubernetes node via hostPath somewhere other than the Kubelet.
+    plugin registry (typically `/var/lib/kubelet/plugins/<drivername.example.com>/csi.sock`).
+  * This is the socket referenced by the `--csi-address` argument.
+
+### Required arguments
+
+None
+
+### Required permissions
+
+The cluster-driver-registrar needs to be able to create and delete CSIDriver
+objects. It also needs to be able to mount hostPath volumes and have the file
+permissions to:
+
+* Access the CSI driver socket (typically in `/var/lib/kubelet/plugins/<drivername.example.com>/`).
+  * Used by the `cluster-driver-registrar` to fetch the driver name from the driver
+    contain (via the CSI `GetPluginInfo()` call).
+
+### Example
+
+Here is an example sidecar spec in the driver Deployment. `<drivername.example.com>` should be replaced by
+the actual driver's name.
+
+```bash
+      containers:
+        - name: cluster-driver-registrar
+          image: quay.io/k8scsi/csi-cluster-driver-registrar:v1.0.2
+          args:
+            - "--csi-address=/csi/csi.sock"
+          volumeMounts:
+            - name: plugin-dir
+              mountPath: /csi
+      volumes:
+        - name: plugin-dir
+          hostPath:
```

Comment:
this can be replaced by emptydir.

I guess in this example, it may make sense to also reference the driver container to show that they're sharing the same emptydir?

Reply:
I think it's obvious enough given the rest of the documentation. I'm not sure what an "example" would look like, is there any reference for that?

Reply:
The external-attacher has an [example](https://github.com/kubernetes-csi/external-attacher/blob/master/deploy/kubernetes/deployment.yaml) but it's pretty long.  I'm fine with leaving out the driver container and just documenting that this emptydir is shared with the driver.

It would be good to document somewhere that this cluster-driver-registrar should be run with the controller portion, and not the per-node portion.